src/pages/protocol/[...protocol].js

```diff
@@ -0,0 +1,52 @@
+import { GeneralLayout } from '../../layout'
+import ProtocolContainer from 'containers/ProtocolContainer'
+
+import { getProtocols, getProtocol } from 'api'
+import { standardizeProtocolName, capitalizeFirstLetter } from 'utils'
+import { ProtocolDataProvider } from 'contexts'
+
+export async function getStaticProps({
+  params: {
+    protocol: [protocol, selectedChain = null, denomination = null]
+  }
+}) {
+  const { protocolsDict } = await getProtocols()
+  const protocolData = await getProtocol(protocol)
+
+  return {
+    props: {
+      protocol,
+      selectedChain,
+      denomination,
+      protocolData: {
+        ...(protocolsDict[protocol] || {}),
```

Comment:
afaik /protocol/x should be a superset of /protocols, so this shouldn't change much, but if it doesn't wouldn't it be better to apply the changes directly to the /protocol lambda?

Reply:
ya i think it would be better to apply it to protocol lambda. Maybe we create a v3 so we don break our current prod.
I rmb there is a slight difference between the two calls. https://github.com/DefiLlama/defillama-app/blob/47429bd76e51d8e72db49749c5f5b53ceb52e1e1/src/contexts/TokenData.js#L112